react-common/components/controls/EmbedVideo.tsx

```diff
@@ -1,45 +1,69 @@
-import * as React from "react";
+
 import { classList, ControlProps } from "../util";
+import { useRef, useEffect } from 'react';
+import * as React from 'react';
+import { MediaPlayer } from "dashjs";
+
 
 export interface EmbedVideoProps extends ControlProps {
-  src: "youtube";
-  id: string;
-  autoplay?: boolean;
-  allowFullScreen?: boolean;
+    src: "youtube" | "azuremedia";
+    id: string;
+    autoplay?: boolean;
+    allowFullScreen?: boolean;
 }
 
-
-
 export const EmbedVideo = (props: EmbedVideoProps) => {
+    let endpointName;
```

Comment:
```suggestion
    const endpointName = "makecode-lucas-testing-makecodetempmediaservice-usea";
```

I believe abhijith just mean to move this to get the temp part of the endpoint name out of the built url, and see below \/